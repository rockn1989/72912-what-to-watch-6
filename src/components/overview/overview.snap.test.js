import React from 'react';
import {render} from '@testing-library/react';
import Overview from './overview';

it(`Should PageNotFound render correctly`, () => {
  const film = {
    starring: [`Man`, `Woman`]
  };

  const {container} = render(<Overview film={film} />);
  expect(container).toMatchSnapshot();
});
