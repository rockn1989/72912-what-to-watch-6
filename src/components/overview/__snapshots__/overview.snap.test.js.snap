// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should PageNotFound render correctly 1`] = `
<div>
  <div
    class="movie-rating"
  >
    <div
      class="movie-rating__score"
    />
    <p
      class="movie-rating__meta"
    >
      <span
        class="movie-rating__level"
      >
        Very good
      </span>
      <span
        class="movie-rating__count"
      >
         ratings
      </span>
    </p>
  </div>
  <div
    class="movie-card__text"
  >
    <p />
    <p
      class="movie-card__director"
    >
      <strong>
        Director: 
      </strong>
    </p>
    <p
      class="movie-card__starring"
    >
      <strong>
        Starring: 
        Man, 
        Woman
      </strong>
    </p>
  </div>
</div>
`;
